Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -531,14 +540,29 @@ void VulkanRenderManager::ThreadFunc() {
 	VLOG("PULL: Quitting");
 }
 
+void VulkanRenderManager::PresentWaitThreadFunc() {
+	SetCurrentThreadName("PresentWait");
+
+	uint64_t waitedId = frameIdGen_;
+	while (run_) {
+		const uint64_t timeout = 1000000000ULL;  // 1 sec
+		vkWaitForPresentKHR(vulkan_->GetDevice(), vulkan_->GetSwapchain(), waitedId, timeout);
```

Comment:
This seems to be really inefficient on my driver, and looks like it's just doing a spin loop - pegging a thread on my driver.  Probably a bad thing on laptops...

-[Unknown]

Reply:
Yeah, obviously not ideal. I'll try something. Also, in MAILBOX mode we probably shouldn't bother (which currently is set by "vsync off")

Reply:
I'm also getting this, it seems this extension isn't really ready for prime time :(  Might have to give up on using it.

Reply:
Seems like it spins calling SwitchToThread(), at least, so maybe it wouldn't hurt when there's other work to do on that core.  Still bad for laptop battery life...

-[Unknown]